src/onedrive.d

```diff
@@ -646,7 +645,6 @@ final class OneDriveApi
 			//	412 - Precondition Failed
 			case 412:
 				throw new OneDriveException(http.statusLine.code, http.statusLine.reason);
-				break;
```

Comment:
This is the only 'break' removal I am concerned about.

What 'should be happening is that the exception is raised, but raised in a manner that sync.d will then handle that the 412 error was thrown

This was added in here https://github.com/abraunegg/onedrive/commit/8031f9491a36bc6aa5d1be3d9267af926a8427b2#diff-bcb5648661675827ec4bb95690ff2754

Potentially the 'throw' itself needs to be replaced with:

```
log.vlog("OneDrive returned a 'HTTP 412 - Precondition Failed' - gracefully handling error");
```